scvi/inference/annotation.py

```diff
@@ -17,32 +17,32 @@
 from scvi.inference.inference import UnsupervisedTrainer
 from scvi.inference.posterior import unsupervised_clustering_accuracy
 
+logger = logging.getLogger(__name__)
+
 
 class AnnotationPosterior(Posterior):
     def __init__(self, *args, model_zl=False, **kwargs):
         super().__init__(*args, **kwargs)
         self.model_zl = model_zl
 
-    def accuracy(self, verbose=False):
+    def accuracy(self):
         model, cls = (self.sampling_model, self.model) if hasattr(self, 'sampling_model') else (self.model, None)
         acc = compute_accuracy(model, self, classifier=cls, model_zl=self.model_zl)
-        if verbose:
-            logging.info("Acc: %.4f" % (acc))
+        logger.debug("Acc: %.4f" % (acc))
```

Comment:
Hey, is there a reason why you use `debug` instead of `info`? It seems to me it's more an information about the model rather than a software engineering debug information.

Reply:
not really, just thought that would be a debugging level information (that we kind of never really want to read unless we have trouble with the model fit)

Reply:
I agree with Romain on this